packages/order-experience-api/src/openapi.yaml

```diff
@@ -633,7 +635,7 @@ paths:
 
   /{orderId}/invoice:
     post:
-      description: Set order's invoice
+      description: Set or update order's invoice - UserId authorization is required
```

Comment:
- UserId authorization is required
 
 I think this is not required to say, because you can use HelsinkiProfile bearer token aswell for authentication. Does those security fields show up in somewhere in swagger?

Reply:
While it's true that you can use HelsinkiProfile bearer token for authentication was well, but the endpoint itself requires value named 'user' to be set in headers for the endpoint to function - this need is fulfilled by the usage of UserId authorization with this endpoint. Setting a header value to be required by swagger results in a double user header value, thus my added description. Either way, it's a bit of a wonky situation from the context Swagger end users.

Reply:
That is not correct, because if you look deeper in withAuthentication wrapper in routes.ts there is append to headers.user field.

 req.headers.user = claims.sub
 verkkokauppa-experience-api/packages/auth-helsinki-profile/src/controller.ts:64